src/teleop/gui/src/components/ArmControls.vue

```diff
@@ -70,8 +129,54 @@ export default {
   methods: {
     updateArmEnabled: function (enabled) {
       this.arm_enabled = enabled;
+      //ra_command, have all nulls in position. And all zeros in velocity array
+      if (this.arm_enabled === false) {
+        const zeroCommandData = {
+          name: [
+            "joint_a",
+            "joint_b",
+            "joint_c",
+            "joint_d",
+            "joint_e",
+            "joint_f"
+          ],
+          position: [null, null, null, null, null, null],
+          velocity: [0, 0, 0, 0, 0, 0],
+          effort: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
+        };
+        var zeroCommandMsg = new ROSLIB.Message(zeroCommandData);
+        this.jointstate_pub.publish(zeroCommandMsg);
+      }
     },
-  },
+    updateServo: function (enabled) {
+      this.servo_enabled = enabled;
+    },
+    updateOpenLoop: function (enabled) {
```

Comment:
I think it might be more readable to put the code of some of these one-liners directly in the html template part instead of calling this function from there